etc/encoding/opencast-movies.properties

```
@@ -196,4 +196,4 @@ profile.editor.work.input = audiovisual
 profile.editor.work.output = audiovisual
 profile.editor.work.suffix = -editor.mp4
 profile.editor.work.mimetype = video/mp4
-profile.editor.work.ffmpeg.command = -i #{in.video.path} -filter:v fps=25 -shortest -c:v libx264 -preset superfast -pix_fmt yuv420p -crf 18 -c:a aac -strict -2 -b:a 196k #{out.dir}/#{out.name}#{out.suffix}
+profile.editor.work.ffmpeg.command = -i #{in.video.path} -filter:v 'crop=trunc(iw/2)*2:trunc(ih/2)*2,fps=25' -shortest -c:v libx264 -preset superfast -pix_fmt yuv420p -crf 18 -c:a aac -strict -2 -b:a 196k #{out.dir}/#{out.name}#{out.suffix}
```

Since this is no shell, shouldn't this just be:

```suggestion
profile.editor.work.ffmpeg.command = -i #{in.video.path} -filter:v crop=trunc(iw/2)*2:trunc(ih/2)*2,fps=25 -shortest -c:v libx264 -preset superfast -pix_fmt yuv420p -crf 18 -c:a aac -strict -2 -b:a 196k #{out.dir}/#{out.name}#{out.suffix}
```